README.md

```diff
@@ -305,6 +306,30 @@ method takes one argument:
 
 2. `chain` must return a value of the same Chain
 
+### ChainRec
+
+A value that implements the ChainRec specification must also implement the Chain specification.
+
+1. `T.chainRec((done, next, value) => g(done(value)), i) `is equivalent to `g(i)`
```

Comment:
That's right. The first law says these two should give equal results, and the second says `chainRec` should do it in no more stack (asymptotically) than used by `f`.